js/articles.js

```diff
@@ -0,0 +1,71 @@
+fetch('jsons/articles.json')
+  .then(response => {
+    return response.json()
+  })
+  .then(data => {
+    // Work with JSON data here
+    articlesCounter = data["articles"].length;
+    let a = Math.floor(Math.random() * articlesCounter);
+    let b = Math.floor(Math.random() * articlesCounter);
+    let c = Math.floor(Math.random() * articlesCounter);
+
+    const width = screen.width;
+    const articles = document.getElementsByClassName("articles")[0];
+    if (width>=800){
+      bildArticle(a);
+      bildArticle(b);
+      bildArticle(c);
+    } else if(width<400){
+      bildArticle(a);
+    } else{
+      bildArticle(a);
+      bildArticle(b);
+    }
+
+    function bildArticle(i){
```

Comment:
space before brackets
and "i" is uninformative name
please, specify what index is it. articleIndex or smth else